src/test/scala/org/camunda/feel/impl/builtin/BuiltinConversionFunctionsTest.scala

```diff
@@ -211,7 +211,13 @@ class BuiltinConversionFunctionsTest
     eval(""" string(date and time("2012-12-25T11:00:00+02:00")) """) should be(
       ValString("2012-12-25T11:00:00+02:00"))
   }
+  it should "convert negative days-time-duration" in {
 
+    eval(""" string(@"-PT1S") """) should be(ValString("PT-1S"))
+    eval(""" string(@"-PT1H") """) should be(ValString("PT-1H"))
+    eval(""" string(@"-PT2M30S") """) should be(ValString("PT-2M-30S"))
+    eval(""" string(@"-P1DT2H3M4S") """) should be(ValString("P-1DT-2H-3M-4S"))
```

Comment:
:x: I had a deeper look at the DMN spec. The expected format is `"-PT1S"` instead of `"PT-1S"`. 

Please see the details [here](https://github.com/camunda/feel-scala/issues/599#issuecomment-1522756437).